### ISSUE TITLE
Why does the connected ORCID iD not show?

### ISSUE THREAD

**Issue by DTMA** [deic-dk/user_orcid#6]
When a user succesfully connects his or hers ORCID iD the numeric string does not show in https://sciencedata.dk/index.php/settings/personal#panel-userapps   

I think it should, because 
1) it would make it evident that the connection actually happend
2) it would enable the user to verify that the connected ID is indeed the correct one (not that I think many actually do that, however, as long as personal ORCIDs are allowed there may just be more than one ORCID ID).

I acknowledge that the user gets a "connection was successful" message, but still. It is just words - showing the numeric string is proof.

**Comment by deltafunction** [comment]
Yes, the number should show and it does for me. I'll investigate.
